static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -164,20 +169,23 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		items["copyid"] = {name : 'Copy PID to Clipboard'};
 		
 		// Admin actions
-		if ($.inArray('editAccessControl', metadata.permissions) != -1) {
+		var adminItems = [];
+		if ($.inArray('destroy', metadata.permissions) != -1 || $.inArray('reindex', metadata.permissions) != -1) {
 			items["sepdestroy"] = "";
-			items["runEnhancements"] = {name : 'Run enhancements'};
-			if ($.inArray('purgeForever', metadata.permissions) != -1) {
+			if ($.inArray('reindex', metadata.permissions) != -1) {
+				items["runEnhancements"] = {name : 'Run enhancements'};
 				items["reindex"] = {name : 'Reindex'};
-				items["destroy"] = {name : 'Destroy', disabled :  $.inArray('Active', metadata.status) != -1};
+			}
+			if ($.inArray('destroy', metadata.permissions) != -1) {
+				items["destroy"] = {name : 'Destroy', disabled :  !metadata.isDeleted};
 			}
 		}
 		
 		// Trash actions
```

Comment:
maybe we should remove the word 'trash' from the comment, since we're no longer using 'trash' in any of our actions?

Reply:
Sure, but i think i'd like to do that in another PR. I'll add changing the language to the list of admin ui changes

Reply:
👍